src/tour-operator-updates/screens/TOUpdates.tsx

```diff
@@ -79,32 +133,34 @@ const TOUpdates = () => {
                     </TableContainer>
                 </Paper>
 
-                <Paper elevation={2} style={{height: 'fit-content'}}>
+                <Paper elevation={2} style={{ height: 'fit-content' }}>
                     <div className='mt-4 mb-6 flex flex-row gap-2 items-center justify-center'>
-                        <Hotel style={{fontSize: 18}}/>
+                        <Hotel style={{ fontSize: 18 }} />
                         <h3 className='text-xl'>Aktualizacje hoteli</h3>
                     </div>
 
                     <TableContainer>
                         <Table>
                             <TableHead>
                                 <TableRow>
-                                    <TableCell className='border border-gray-300 p-2'>Data</TableCell>
+                                    <TableCell className='border border-gray-300 p-2'>Data zmiany</TableCell>
                                     <TableCell className='border border-gray-300 p-2'>Hotel</TableCell>
                                     <TableCell className='border border-gray-300 p-2'>Pokój</TableCell>
                                     <TableCell className='border border-gray-300 p-2'>Zmiany</TableCell>
+                                    <TableCell className='border border-gray-300 p-2'>Typ zmiany</TableCell>
                                 </TableRow>
                             </TableHead>
                             <TableBody>
-                                {hotelChanges.map((hotel, index) => (
+                                {hotelUpdates.map((hotel, index) => (
                                     <TableRow key={index}>
-                                        <TableCell className='border border-gray-300 p-2'>{hotel.date.toDateString()}</TableCell>
+                                        <TableCell className='border border-gray-300 p-2'>{new Date(hotel.updateDateTime).toLocaleString()}</TableCell>
                                         <TableCell className='border border-gray-300 p-2'>{hotel.hotelName}</TableCell>
                                         <TableCell className='border border-gray-300 p-2'>{hotel.roomName}</TableCell>
+                                        <TableCell className='border border-gray-300 p-2'>{hotel.updateType}</TableCell>
                                         <TableCell className='border border-gray-300 p-2'>
                                             <div className='flex flex-col gap-1'>
-                                                <p>Cena: {hotel.changes.priceDiff >= 0 ? '+' : ''}{hotel.changes.priceDiff}</p>
-                                                <p>Miejsca: {hotel.changes.guestCapacityDiff >= 0 ? '+' : ''}{hotel.changes.guestCapacityDiff}</p>
+                                                <p>Cena: {hotel.priceChange > 0 ? '+' : ''}{hotel.priceChange}</p>
+                                                <p>Miejsca: {hotel.capacityChange > 0 ? '+' : ''}{hotel.capacityChange}</p>
```

Comment:
jeżeli typ zmiany to `CREATE` to nie wyświetlajmy zer, może pustą komórkę?

Reply:
Oki zmienione:
![image](https://github.com/Microarchitecturovisco/travel-ui/assets/72522808/51394ef8-1599-470e-b96a-9676dba02011)